src/App.tsx

```diff
@@ -16,8 +17,10 @@ function App() {
           <Route path="/*" element={<MainPage />} />
           <Route path="/signin" element={<Signin />} />
           <Route path="/signup" element={<Signup />} />
-          <Route path="/myplaylist" element={<MyPlayList />} />
-          <Route path="/myplaylist/:id" element={<MyPlayListDetail />} />
+          <Route path="/myplaylist" element={<AuthRoute />}>
+            <Route path="/myplaylist" element={<MyPlayList />} />
+            <Route path="/myplaylist/:id" element={<MyPlayListDetail />} />
+          </Route>
```

Comment:
헤더에서 myplaylist 접속시 로그인 여부 판단해서  navigate('/signin') 이렇게 지정을 해주면
라우팅은 이 부분만 있으면 될 거 같아요
```suggestion
<Route path="/myplaylist" element={<MyPlayList />} />
<Route path="/myplaylist/:id" element={<MyPlayListDetail />} />
```

Reply:
근데 저런식으로 하면 url로 직접접근이 가능해서요..!

Reply:
음 어떤 url로 접근하려는 건가요?